awx/ui/client/features/credentials/edit-credentials.controller.js

```diff
@@ -43,10 +43,11 @@ function EditCredentialsController (models, $state, $scope, strings, componentsS
     }
 
     const isOrgAdmin = _.some(me.get('related.admin_of_organizations.results'), (org) => org.id === organization.get('id'));
+    const isOrgCredentialAdmin = organization.search({ role_level: 'credential_admin_role' }).then((data) => data);
```

Comment:
Since this is a promise, I'm wondering if it might be better to do the search as a resolvable item in this view's route / state configuration? https://github.com/ansible/awx/blob/devel/awx/ui/client/features/credentials/index.js